lib/orderbook/OrderBook.ts

```diff
@@ -345,6 +352,53 @@ class OrderBook extends EventEmitter {
     return outgoingOrder;
   }
 
+  /**
+   * Handles a request from a peer to create a swap deal. Checks if the order for the requested swap
+   * is available and if a route exists to determine if the request should be accepted or rejected.
+   * Responds to the peer with a swap response packet containing either an accepted quantity or rejection reason.
+   */
+  private handleSwapRequest = async (requestPacket: SwapRequestPacket, peer: Peer)  => {
+    assert(requestPacket.body, 'SwapRequestPacket does not contain a body');
+    assert(this.swaps, 'swaps module is disabled');
```

Comment:
I started to change this but in thinking about it a bit more, the *only* time that the swaps module would be disabled is when running unit/integration tests where the p2p module is also disabled and we can't receive swap requests anyhow. `this.swaps` will always be initialized when running  an actual `xud` instance. I put this in just as a sanity check, and I'm thinking actually we'd probably want the crash should this assertion ever fail. I could leave this assertion out entirely, but it will crash anyway due to trying to read properties of an `undefined` field later on. Thoughts?

Reply:
What actually probably does need to be changed is the assert for `requestPacket.body` - since that could definitely happen with a running instance. Maybe I'll just drop that line? It should be handled by future changes to verify the structure and integrity of received packets.